modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -296,20 +296,60 @@ def min(self, **kwargs):
         return self._agg("min", **kwargs)
 
     def sum(self, **kwargs):
+        min_count = kwargs.pop("min_count")
+        if min_count != 0:
+            return super().sum(
+                min_count=min_count,
+                **kwargs,
+            )
         return self._agg("sum", **kwargs)
 
     def mean(self, **kwargs):
         return self._agg("mean", **kwargs)
 
+    def nunique(self, axis=0, dropna=True):
+        if axis != 0 or not dropna:
+            return super().nunique(axis=axis, dropna=dropna)
+        return self._agg("count", distinct=True)
+
     def _agg(self, agg, axis=0, level=None, **kwargs):
+        """
+        Perform specified aggregation along rows/columns.
+
+        Parameters
+        ----------
+        agg : str
+            Name of the aggregation function to perform.
+        axis : {0, 1}, default: 0
+            Axis to perform aggregation along. 0 is to apply function against each column,
+            all the columns will be reduced into a single scalar. 1 is to aggregate
+            across rows.
+            *Note:* OmniSci backend supports aggregation for 0 axis only, aggregation
+            along rows will be defaulted to pandas.
+        level : None
+            Serves the compatibility purpose, always have to be None.
+        **kwargs : dict
+            Additional parameters to pass to the aggregation function.
+
+        Returns
+        -------
+        DFAlgQueryCompiler
+            New single-column (``axis=1``) or single-row (``axis=0``) query compiler containing
+            the result of aggregation.
+        """
         if level is not None or axis != 0:
             return getattr(super(), agg)(axis=axis, level=level, **kwargs)
 
-        skipna = kwargs.get("skipna", True)
-        if skipna is False:
+        # TODO: Do filtering on numeric columns if `numeric_only=True`
+        if (
+            kwargs.get("skipna", True) is not None and not kwargs.get("skipna", True)
```

Comment:
@dchigarev , @vnlitvinov , do you really think that `kwargs.get("skipna", True) is not None and not kwargs.get("skipna", True)` is better than `kwargs.get("skipna", True) is False`?

Reply:
At least it covers all of the cases.

And also, this is only a "temporary" solution until #3195 will be resolved.